packages/blade/src/components/Modal/Modal.native.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { ModalHeader } from './ModalHeader';
+import type { ModalHeaderProps } from './ModalHeader';
+import { ModalFooter } from './ModalFooter';
+import type { ModalFooterProps } from './ModalFooter';
+import { ModalBody } from './ModalBody';
+import type { ModalBodyProps } from './ModalBody';
+import { Text } from '~components/Typography';
+
+type ModalProps = {
```

Comment:
Let's remove or reexport this type from a single place?

Reply:
I've added this just to avoid TS errors at build time. This is a non-critical file and might be pointless to maintain this in single place too. I'll add a comment